content/en/docs/started/installing-kubeflow.md

```diff
@@ -75,6 +75,14 @@ Packaged distributions are developed and supported by their respective maintaine
         <td><a href="/docs/distributions/openshift/">Docs</a></td>
         <td></td>
       </tr>
+      <tr>
```

Comment:
I believe that adding this new distribution on top of the list is not a best practice.

Please change to alphabetical order.

Reply:
Thanks @RFMVasconcelos  
I added to the last of "Kubeflow on" names so that table looks neat. No preference other than that. Preview Link: https://deploy-preview-2865--competent-brattain-de2d6d.netlify.app/docs/started/installing-kubeflow/#install-a-packaged-kubeflow-distribution

Currently, I don't see the table ordered alphabetically.  Do you mean, order the whole table alphabetically in this PR? Or Move this item alone to the last?